packages/plugin-inline/FormatDomParser.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractParser } from '../core/src/AbstractParser';
+import { VNode } from '../core/src/VNodes/VNode';
+import { InlineNode } from '../plugin-inline/InlineNode';
+import { Format } from './Format';
+
+export abstract class FormatDomParser extends AbstractParser<Node> {
+    static id = 'dom';
+
+    /**
+     * Parse a span node.
+     *
+     * @param nodes
+     */
+    applyFormat(format: Format, nodes: VNode[]): void {
```

Comment:
Well now that I think about it, this function might go into an abstract `FormatParser` rather than `FormatDomParser`, because it has nothing to do with DOM, it operates only on our virtual representation. So basically, I'd rename this class into `FormatParser` and put back the `static id` in the different parsers.